service/worker/replicator/processor_test.go

```diff
@@ -320,56 +319,6 @@ func (s *replicationTaskProcessorSuite) TestDecodeMsgAndSubmit_SyncActivity_Fail
 	s.processor.decodeMsgAndSubmit(s.mockMsg)
 }
 
-func (s *replicationTaskProcessorSuite) TestDecodeMsgAndSubmit_History_Success() {
```

Comment:
you might want to simple turn the message from REPLICATION_TASK_TYPE_HISTORY_TASK -> REPLICATION_TASK_TYPE_HISTORY_TASK_V2

Reply:
Filed task to add this later #710